comixed-frontend/src/assets/i18n/pt/app.json

```diff
@@ -316,136 +316,136 @@
   },
   "authentication-effects": {
     "submit-login-data": {
       "success": {
-        "detail": "Successfully logged in!"
+        "detail": "Entrada com sucesso!"
       },
       "failure": {
-        "detail": "Login failed..."
+        "detail": "Erro na entrada..."
       }
     },
     "logout": {
-      "detail": "Successfully logged out."
+      "detail": "Saída com sucesso."
     }
   },
   "breadcrumb": {
     "collections": {
-      "root": "Collections",
-      "publishers": "Publishers",
+      "root": "Coleções",
+      "publishers": "Editoras",
       "series": "Series",
-      "characters": "Characters",
-      "teams": "Teams",
-      "locations": "Locations",
-      "stories": "Stories",
-      "lists": "Reading Lists"
+      "characters": "Personagens",
+      "teams": "Equipas",
+      "locations": "Localizações",
+      "stories": "Historias",
+      "lists": "Listas de leitura"
     },
     "entry": {
       "library-page": "Comics",
-      "duplicates-page": "Duplicates",
+      "duplicates-page": "Duplicados",
       "comic-details-page": {
         "publisher": "{name}",
         "series": "{name}",
         "volume": "v{volume}",
-        "issue-number": "Issue #{issue}"
+        "issue-number": "Número #{issue}"
       },
-      "multi-comic-scraping-page": "Scraping",
-      "account-page": "Account",
+      "multi-comic-scraping-page": "Obter metadados",
+      "account-page": "Conta",
       "help": {
-        "root": "Help",
-        "build-details-page": "Build Details"
+        "root": "Ajuda",
+        "build-details-page": "Informação da compilação"
       },
       "admin": {
-        "root": "Admin",
-        "library-admin": "Library",
+        "root": "Administrador",
+        "library-admin": "Biblioteca",
         "plugins-page": "Plugins",
-        "users-admin": "Users",
-        "import-page": "Import Comics",
-        "duplicates-page": "Duplicates Page",
-        "missing-comics": "Missing Comics",
-        "task-audit-log": "Task Log",
-        "rest-audit-log": "REST logs"
+        "users-admin": "Utilizador",
+        "import-page": "Importar comics",
+        "duplicates-page": "Paginas duplicadas",
+        "missing-comics": "Comics em falta",
+        "task-audit-log": "Registro de tarefas",
+        "rest-audit-log": "Registro REST"
       }
     }
   },
   "general-message": {
     "error": {
-      "general-service-failure": "A general system failure has occurred."
+      "general-service-failure": "Aconteceu um erro genérico."
     }
   },
   "global": {
     "errors": {
-      "form-field-required": "{name} is required."
+      "form-field-required": "{name} é necessário."
     },
     "tooltip": {
-      "unprocessed-comic": "This comic is scheduled for processing.",
-      "deleted-comic": "This comic is marked for deletion."
+      "unprocessed-comic": "Este comic está agendado para ser processado.",
+      "deleted-comic": "Este comic está selecionado para ser apagado."
     }
```

Comment:
I found a missing translation from #376 that wasn't added to this file. Can you add the following to the block below tooltip and translate it, please?

```
    "confirmation": {
      "logout": {
        "header": "Logout",
        "message": "Are you sure you want to log out?"
      }
    }
```

Reply:
Added to the file